web/controllers/friends_controller.ex

```diff
@@ -5,36 +5,57 @@ defmodule Entice.Web.FriendsController do
 
   plug :ensure_login
 
+  #Case to think about:
+  #Player A adds friend Player B under name N
+  #Player B deletes account (and so all chars are deleted)
+  #Player C creates acccount and char under name N
+  #Player A adds Player C under name N
+  #Now in friends DB: (N, PlayerA.id, PlayerB.id) and (N, PlayerA.id, PlayerC.id)
+  #Since all queries are done by name it'll be an issue.
+  #Solutions:
+  #Delete all friends with PlayerB.id in them
+
   @doc "Returns all friends of connected account."
   def index(conn, _params) do
     id = get_session(conn, :client_id)
     {:ok, friends} = Entice.Web.Client.get_friends(id)
 
-    friends = for friend <- friends, do: friend.friend_account.id
-    conn |> json ok%{
+    friends = for friend <- friends do
+      {:ok, status, name} = get_status(friend.basename)
+      if name != friend.basename do
+        "#{friend.basename} (#{name}) [#{status}]"
```

Comment:
this is an api route, i think we dont want to render strings here, but rather put it into a json format